plantcv/plantcv/annotate/classes.py

```diff
@@ -0,0 +1,169 @@
+
+        Returns:
+        :param coords: list
+        :param label: string
+        :return:
+        """
+        if label not in self.points:
+            self.points[label] = []
+            for (y, x) in coords:
+                self.points[label].append((x, y))
+            self.count[label] = len(self.points[label])
+
+        else:
+            warn(f"{label} already included and counted, nothing is imported!")
+
+    def save_coords(self, coord_file):
+        """Save collected coordinates to a file.
+        Input variables:
+        coord_file = Name of the file to save collected coordinate
+        :param coord_file: str
+        """
+        # Open the file for writing
+        with open(coord_file, "w") as fp:
+            # Save the data in JSON format with indentation
+            json.dump(obj=self.points, fp=fp, indent=4)
+
+    def view(self, label="total", color="c", view_all=False):
+        """
+        View the label for a specific class label
+        Inputs:
+        label = (optional) class label, by default label="total"
+        color = desired color, by default color="c"
+        view_all = indicator of whether view all classes, by default view_all=False
+
+        :param label: string
+        :param color: string
+        :param view_all: boolean
+        :return:
+        """
+        if label not in self.points and color in self.colors.values():
+            warn("The color assigned to the new class label is already used, if proceeding, "
+                 "items from different classes will not be distinguishable in plots!")
+
+        self.label = label
+        self.color = color
+        self.view_all = view_all
+
+        if label not in self.points:
+            self.points[label] = []
+            self.count[label] = 0
+        self.colors[label] = color
+
+        self.fig, self.ax = plt.subplots(1, 1, figsize=self.figsize)
+
+        self.events = []
+        self.fig.canvas.mpl_connect('button_press_event', self.onclick)
+
+        self.ax.imshow(cv2.cvtColor(self.img, cv2.COLOR_BGR2RGB))
+        self.ax.set_title("Please left click on missing pollens\n Right click on those you want to remove")
+        self.p_not_current = 0
+        # if view_all is True, show all already marked markers
+        if view_all:
+            for k in self.points:
+                for (x, y) in self.points[k]:
+                    self.ax.plot(x, y, marker='x', c=self.colors[k])
+                    if self.label not in self.points or len(self.points[self.label]) == 0:
+                        self.p_not_current += 1
+        else:
+            for (x, y) in self.points[label]:
+                self.ax.plot(x, y, marker='x', c=color)
+
+    def onclick(self, event):
+        """Handle mouse click events."""
+        self.events.append(event)
+        if event.button == 1:
+            self.ax.plot(event.xdata, event.ydata, marker='x', c=self.color)
+            self.points[self.label].append((event.xdata, event.ydata))
```

Comment:
I think this needs updating to resolve the error with `pcv.roi.multi`. Compare to line 54 of this file, i think we just need to wrap it in a `floor()` function.